ts/IterableInboxMessageDisplay.tsx

```diff
@@ -7,25 +7,29 @@ import {
   ScrollView,  
   StyleSheet,
   Platform,
+  Alert,
```

Comment:
I dont see Alert being used. You can remove unused imports.

Reply:
Yes. Alert was being used for debugging. I will remove.